addons/website_event/static/tests/tours/registered_banner.js

```diff
@@ -0,0 +1,65 @@
+/** @odoo-module **/
+
```

Comment:
fyi no need to add this for files under `/static/src` and `/static/tests` (see `is_odoo_module` in `js_transpiler.py`)